examples/lighting-app/efr32/README.md

```diff
@@ -145,18 +166,87 @@ combination with JLinkRTTClient as follows:
 
 ## Running the Complete Example
 
--   Once the example is flashed on the board, you should be able to establish a
-    connection with an OpenThread border router. See
+-   It is assumed here that you already have a OpenThread border router
+    configured and running. If not See the following guide
     [OpenThread Border Router](https://openthread.io/guides/border-router) for
     more information on how to setup a border router. Take note that the RCP
     code is available directly through
     [Simplicity Studio 5](https://www.silabs.com/products/development-tools/software/simplicity-studio/simplicity-studio-5)
     under File->New->Project Wizard->Examples->Thread : ot-rcp
--   Once said connectection is established (you can verify that with the command
+
+-   User interface : **LCD** The LCD on Silabs WSTK shows a QR Code. This QR
+    Code is be scanned by the CHIP Tool app For the Rendez-vous procedure over
+    BLE
+
+        * On devices that do not have or support the LCD Display like the BRD4166A Thunderboard Sense 2,
+          a URL can be found in the RTT logs.
+
+          <info  > [SVR] Copy/paste the below URL in a browser to see the QR Code:
+          <info  > [SVR] https://dhrishi.github.io/connectedhomeip/qrcode.html?data=CH%3AI34NM%20-00%200C9SS0
+
+    **LED 0** shows the overall state of the device and its connectivity. The
+    following states are possible:
+
+        -   _Short Flash On (50 ms on/950 ms off)_ ; The device is in the
+            unprovisioned (unpaired) state and is waiting for a commissioning
+            application to connect.
+
+        -   _Rapid Even Flashing_ ; (100 ms on/100 ms off)_ &mdash; The device is in the
+            unprovisioned state and a commissioning application is connected through
+            Bluetooth LE.
+
+        -   _Short Flash Off_ ; (950ms on/50ms off)_ &mdash; The device is fully
+            provisioned, but does not yet have full Thread network or service
+            connectivity.
+
+        -   _Solid On_ ; The device is fully provisioned and has full Thread
+            network and service connectivity.
+
+    **LED 1** Simulates the Light The following states are possible:
+
+        -   _Solid On_ ; Light is on
+        -   _Off_ ; Light is off
+
+    **Push Button 0** - Press and Release : If not commissionned, start thread
+    with default configurations (DEBUG)
+
+
+        -   Pressed and hold for 6 s: Initiates the factory reset of the device.
+            Releasing the button within the 6-second window cancels the factory reset
+            procedure. **LEDs** blink in unison when the factory reset procedure is
+            initiated.
+
+    **Push Button 1**
+        Toggles the light state On/Off
+
+-   Once the device is provisonned and it will join the Thread network is
```

Comment:
```suggestion
-   Once the device is provisioned and it will join the Thread network is
```
I can't make sense of the rest of the sentence though.  Once a device is provisioned and it will join the Thread network... then what?  Does "is established" mean to be "it established"?  Or something else?